lib/geordi/gitpt.rb

```diff
@@ -2,12 +2,14 @@ class Gitpt
   require 'yaml'
   require 'highline'
   require 'tracker_api'
+  require 'geordi/settings'
```

Comment:
Please require settings just like in cucumber.rb. While this ^ seems to work, it will require the settings of the Geordi in Ruby's load path, which is not always the same file as the settings file relative to this one.